go.mod

```diff
@@ -28,10 +28,10 @@ require (
 	github.com/evanphx/json-patch v4.1.0+incompatible
 	github.com/fsnotify/fsnotify v1.4.7
 	github.com/fsouza/fake-gcs-server v0.0.0-20180612165233-e85be23bdaa8
-	github.com/go-openapi/jsonpointer v0.0.0-20170102174223-779f45308c19 // indirect
```

Comment:
Were these edits manual?

Reply:
No, i didn't touch this file by hand. Should I reset it to master?

Reply:
If you don't need dep changes yes lets not commit any here